container-scan/action.yaml

```diff
@@ -1,152 +1,74 @@
-name: "Security Scan Composite Action"
-description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+name: Container Security Scan
+description: Scans container images for vulnerabilities using Lacework
 
 inputs:
-  dockerhub-user:
-    required: false
-    default: ""
-    description: username for dockerhub
-  dockerhub-password:
-    required: false
-    default: ""
-    description: password for dockerhub
-  docker-config-file:
-    required: false
-    description: Path to the docker config file (defaults to .docker-config.json) Must contain imageName, may contain dockerfile
-    default: .docker-config.json
-  github-token:
-    required: true
-    description: GitHub token
-  lw-account-name:
-    required: true
-    description: Lacework account name
-  lw-access-token:
-    required: true
-    description: Lacework access token
   image-name:
     required: false
     description: Docker image name
   image-tag:
     required: true
     description: Docker image tag
-  image-platform:
-    description: Target platform to build image for (eg. linux/amd64 (default), linux/arm64, etc)
+  enable-docker-build:
     required: false
-    default: linux/amd64
-
+    default: "true"
+    description: Enable Docker build
   build-args:
     required: false
-    description: List of build arguments for docker build as key-value pairs (e.g., KEY=VALUE)
-    default: ""
+    description: List of build arguments for docker build as key-value pairs
+  docker-config-file:
+    required: false
+    description: Path to the docker config file (defaults to .docker-config.json) Must contain imageName, may contain dockerfile
+    default: .docker-config.json
   secrets:
     required: false
     description: List of secrets for docker build as key-value pairs (e.g., SECRET_KEY=VALUE)
-    default: ""
-  enable-docker-build:
-    required: false
-    default: true
-    description: Docker image tag
+  lw-account-name:
+    required: true
+    description: Lacework account name
+  lw-access-token:
+    required: true
+    description: Lacework access token
 
 outputs:
-  comment-id:
-    description: Comment ID of the test report
-    value: ${{ steps.comment-pr.outputs.comment-id }}
+  scan-results-path:
+    description: Path to the scan results file
+    value: ${{ steps.scan.outputs.results-file }}
 
 runs:
   using: composite
   steps:
     - name: Checkout Repository
       uses: actions/checkout@v4
 
-    - name: Build docker image
-      if: ${{ inputs.enable-docker-build }}
+    - name: Build Docker Image
+      if: ${{ inputs.enable-docker-build == 'true' }}
       uses: open-turo/actions-security/docker-build@v3
       id: docker-build
       with:
-        dockerhub-user: ${{ inputs.dockerhub-user }}
-        dockerhub-password: ${{ inputs.dockerhub-password }}
-        github-token: ${{ inputs.github-token }}
```

Comment:
Maybe also not remove the `github-token` variable? Since it's an input in the docker-build action, it's needed so it can be passed through